### ISSUE TITLE
vim-go func snippet causes errors

### ISSUE THREAD

**Issue by piersy** [neoclide/coc-snippets#93]
The vim-go func snippet is causing an error when I try to expand it. 

Minimal vimrc using vim-plug
```vim
call plug#begin()
Plug 'SirVer/ultisnips'
Plug 'neoclide/coc.nvim', {'branch': 'release'}
Plug 'fatih/vim-go'
call plug#end()
```
Minimal coc config
```json
{
	"snippets.ultisnips.directories": [
		"UltiSnips",
		"gosnippets/UltiSnips"
	]
}
```

Coc info
```
## versions

vim version: NVIM v0.4.3
node version: v12.2.0
coc.nvim version: 0.0.74-15f685206a
term: xterm-256color
platform: linux

## Messages

## Output channel: snippets
```
The competion popup menu before insertion
![coc-snippet-fail](https://user-images.githubusercontent.com/5087847/71723361-4c2ded00-2e24-11ea-87b7-848148f155a5.png)

The text inserted if I press enter
```go
package main

func name(params)Traceback (most recent call last):  
File "<string>", line 3, in <module>
File "/tmp/coc-ultisnips-166c49187976bd49e3cfa0cbc473f03c.py", line 279, in opening_par
if return_values(t[pos]) > 1 and not t[pos].startswith("("):
IndexError: tuple index out of rangeTraceback (most recent call last):
File "<string>", line 3, in <module>
File "/tmp/coc-ultisnips-166c49187976bd49e3cfa0cbc473f03c.py", line 285, in closing_par
if return_values(t[pos]) > 1:
IndexError: tuple index out of range {
        
}

```

**Comment by nicolas-martin** [comment]
Also seeing the results on my end

![image](https://user-images.githubusercontent.com/4259729/74860428-9d336b00-5316-11ea-97f1-3c3f98ba00be.png)

**Comment by chemzqm** [comment]
Fixed on v2.2.6
